src/main/java/org/spdx/maven/CreateSpdxMojo.java

```diff
@@ -93,7 +98,7 @@
  */
 @Mojo( name = "createSPDX",
        defaultPhase = LifecyclePhase.VERIFY,
-       requiresDependencyResolution = ResolutionScope.TEST )
+       requiresDependencyResolution = ResolutionScope.COMPILE )
```

Comment:
Currently, we're including the test dependencies with a relationship type `TEST_DEPENDENCY_OF`.  We should probably leave this as `TEST` since `TEST` will include the `COMPILE` dependencies.

Reply:
Ups, sorry. I changed this when trying to make the test work and forgot to move it back. Done!